packages/@vue/cli/bin/vue.js

```diff
@@ -230,6 +235,17 @@ if (!process.argv.slice(2).length) {
   program.outputHelp()
 }
 
+function suggestCommands (cmd) {
+  const availableCommands = program.commands.map(cmd => {
+    return cmd._name
+  })
+
+  const suggestion = didYouMean(cmd, availableCommands)
+  if (suggestion) {
+    console.log(chalk.cyan(`Warning: ${chalk.yellow(`Did you mean ${suggestion} ?`)}`))
```

Comment:
1. the space before the question mark needs to be removed.
2. should highlight the command name, as in the `Unknown command …` message. Also the `Warning: ` part seems unnecessary considering the overall coloring.